ui/components/FluxRuntime.tsx

```diff
@@ -5,27 +5,102 @@ import Flex from "../components/Flex";
 import { Crd, Deployment } from "../lib/api/core/types.pb";
 import ControllersTable from "./ControllersTable";
 import CrdsTable from "./CrdsTable";
+import FluxVersionsTable, { FluxVersion } from "./FluxVersionsTable";
+import { routeTab } from "./KustomizationDetail";
 import SubRouterTabs, { RouterTab } from "./SubRouterTabs";
+import Text from "./Text";
+
+const FluxVersionText = styled(Text)`
```

Comment:
💡 `Text` is already a styled `span` - you could save some code by getting rid of the span tag in the JSX and some of the styling here. Font weight and color can be done through the props (which I think you might already be doing, in which case this is duplicated? Not sure), but the styled component is good for the margins

Reply:
I thought about adding a two Text as 
```
<Text color="neutral30" titleHeight={true}>
     This cluster is running Flux version:
     <Text color="neutral40" >{deployments[0].labels[fluxVersionLabel]}</Text>
 </Text>
```
& add styles to both of them by adding classes to each of them, in return the lines i'll save by using TEXT will be added by creating classes :thought_balloon: